test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,10 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44,10)).toEqual('I should be there in 4.5 hours');
+    expect(reachDestination(46,10)).toEqual('I should be there in 5 hours');
+    expect(reachDestination(92,10)).toEqual('I should be there in 9.5 hours');
```

Comment:
I love that you have added more tests. It would be good to change the value of 10 as well so that you know that the function works with an argument that isn't 10 for the second parameter.